scripts/vlm/qwen2vl_convert_ckpt.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
```

Comment:
it doesnt need to be a stand alone scirpt. You can put this in doc and also you can put this in the comments of finetune script. Try to keep the folder clear.